src/request-pipeline/xhr/headers.js

```diff
@@ -9,5 +9,6 @@ export default {
     corsSupported:           'x-hammerhead|xhr|cors-supported-header',
     withCredentials:         'x-hammerhead|xhr|with-credentials-header',
     origin:                  'x-hammerhead|xhr|origin',
```

Comment:
Also we need to create separate issues for fetch, image, script and style requests